docs/events/workshops.md

```diff
@@ -0,0 +1,34 @@
+# Workshops
+
+Workshops are an important part of the RCOS experience. They strengthen the presenters communication skills and inform the audience about a skill or technology 
+pertinent to their open source work. 
+
+## Hosting a workshop
+Anyone in RCOS is allowed to host a workshop. Mentors are required to host at least one. 
+
+Workshops have historically be hosted mostly on Wednesdays, since most Computer Science majors at RPI are free on Wednesdays afternoon. Despite this, 
```

Comment:
"Wednesdays afternoon" should probably be "Wednesday afternoons"

Reply:
Fixed!